conf/index-writers.xml.template

```diff
@@ -108,8 +108,10 @@
     <parameters>
       <param name="host" value="localhost"/>
       <param name="port" value="9200"/>
-      <param name="cluster" value=""/>
       <param name="index" value="nutch"/>
+      <!--Configure username and password if necessary, default user is "elastic"
+      <param name="user" value="user"/>
```

Comment:
Nit: Perhaps we should use `username` instead of `user`. For the Solr indexer configuration, we use `username` and also the comment mentions `username`.

Reply:
Nit: perhaps here we should use as well a default `elastic` value for the username?